.github/workflows/test.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ["2.7", "3.0", "3.1", "3.2"]
+        ruby-version: ["3.0", "3.1", "3.2", "3.3", "3.4"]
```

Comment:
@dim I'm dropping ruby v2. Too much hassle with the deps (mostly dry-XXX indirect deps), there are few dry-XXX packages and their latest versions are requiring ruby v3, I don't want to list and override (downgrade) versions for all of them.

Reply:
Btw, I don't know why we enumerate all the minor Ruby versions there + how many last versions should we aim to support, so enumerated all the 3.x available as of today.